CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 - Implement `FromStr` and `ToString` traits for public sodium types (#318)
+- Base support for metrics infrastructure.
```

Comment:
Please be as specific as possible in a changelog. Something like: "Added new macro `metric!` to collect statistical information."